Tribute.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <div id = "img-div">
+            <img id = "image" src="./IMAGES/tribute-image1.jpg" alt="">
+            <div id = "img-caption">
```

Comment:
- Adding spaces between equal signs is not part of HTML Best practices, Kindly remove these spaces